core/src/main/java/org/apache/iceberg/rest/Endpoint.java

```diff
@@ -70,6 +85,7 @@ public class Endpoint {
   public static final Endpoint V1_DELETE_VIEW = Endpoint.create("DELETE", ResourcePaths.V1_VIEW);
   public static final Endpoint V1_RENAME_VIEW =
       Endpoint.create("POST", ResourcePaths.V1_VIEW_RENAME);
+  public static final Endpoint V1_VIEW_EXISTS = Endpoint.create("HEAD", ResourcePaths.V1_VIEW);
```

Comment:
I'll open a separate PR to actually do a `HEAD` request when checking if a view exists

Reply:
https://github.com/apache/iceberg/pull/11760